airbyte-ci/connectors/connector_ops/tests/test_utils.py

```diff
@@ -30,17 +30,17 @@ def test__get_type_and_name_from_technical_name(self, technical_name, expected_t
         "connector, exists",
         [
             (utils.Connector("source-faker"), True),
-            (utils.Connector("source-notpublished"), False),
+            (utils.Connector("source-doesnotexist"), False),
```

Comment:
@alafanechere I think given where the code base is at, we should change this to an error

I.e. if you try to instantiate `Connector` but we cant find a metadata.yaml file. we throw an Error.

What are your thoughts?

Reply:
I'm in for this but I think this could break a couple of test.
I think that in some test scenario we provision a fake connector, without a metadata (or other file).
Feel free to make this change but please run the `pipelines` test suite to make sure nothing breaks. (I believe it will run automatically thanks to your recent change on the workflow).

Reply:
Will do!